WordPress/Classes/ViewRelated/Notifications/Controllers/NotificationsViewController/NotificationsTableHeaderView.swift

```diff
@@ -0,0 +1,73 @@
+import UIKit
+import DesignSystem
+
+final class NotificationsTableHeaderView: UITableViewHeaderFooterView {
+
+    static let reuseIdentifier: String = String(describing: NotificationsTableHeaderView.self)
+
+    // MARK: - Properties
+
+    var text: String? {
+        didSet {
+            self.update(text: text)
+        }
+    }
+
+    // MARK: - Init
+
+    override init(reuseIdentifier: String?) {
+        super.init(reuseIdentifier: reuseIdentifier)
+        self.setup()
+    }
+
+    required init?(coder: NSCoder) {
+        fatalError("init(coder:) has not been implemented")
+    }
+
+    // MARK: - Setup
+
+    private func setup() {
+        self.contentConfiguration = {
+            var config = super.defaultContentConfiguration()
+            config.textProperties.font = Appearance.textFont
+            config.textProperties.color = Appearance.textColor
+            config.directionalLayoutMargins = Appearance.layoutMargins
+            return config
+        }()
+        if #available(iOS 16.0, *) {
+            self.backgroundConfiguration = {
+                var config = self.defaultBackgroundConfiguration()
+                config.backgroundColor = Appearance.backgroundColor
+                config.visualEffect = nil
+                return config
+            }()
+        } else {
+            self.contentView.backgroundColor = Appearance.backgroundColor
+        }
```

Comment:
Please make sure that section header background is a solid white on both iOS 15, and iOS 16+